src/elm/Main.elm

```diff
@@ -4,7 +4,6 @@ import App.Model exposing (Model)
 import App.Router exposing (..)
 import App.Update exposing (init, update, Msg)
 import App.View exposing (view)
-import Html.App as Html
```

Comment:
Yes,  if not used you can remove
On Jun 23, 2016 5:47 PM, "Nader Safadi" <notifications@github.com> wrote:

> In src/elm/Main.elm
> <https://github.com/Gizra/elm-spa-example/pull/17#discussion_r68248411>:
>
> > @@ -4,7 +4,6 @@ import App.Model exposing (Model)
> >  import App.Router exposing (..)
> >  import App.Update exposing (init, update, Msg)
> >  import App.View exposing (view)
> > -import Html.App as Html
>
> @amitaibu <https://github.com/amitaibu> elm-format recommended removing
> this as it's not being used but seems weird to me to remove it, Is it ok ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Gizra/elm-spa-example/pull/17/files/bcaccbea25a20361b4c02289eea9d2a47f155503#r68248411>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHrCxo8jp3bD9nTnlO9XFXRg33cZlirks5qOpxxgaJpZM4I7_oo>
> .
>